acceptance_tests/features/steps/rh_ui.py

```diff
@@ -83,3 +88,18 @@ def error_section_displayed(context, href_name, expected_text):
     test_helper.assertEqual(context.browser.find_by_id('alert').text, 'There is a problem with this page')
     error_text = context.browser.links.find_by_href(href_name).text
     test_helper.assertEqual(error_text, expected_text)
+
+
+@retry(wait=wait_fixed(1), stop=stop_after_delay(30))
+def retry_launch_eq(rh_launch_uac):
+    rh_launch_endpoint_response = rh_endpoint_client.post_to_launch_endpoint(rh_launch_uac)
+    rh_launch_endpoint_response.raise_for_status()
+
+    return rh_launch_endpoint_response
+
+
+@step("check UAC is in firestore via eqLaunched endpoint")
+def check_uac_in_firestore(context):
+    context.rh_launch_endpoint_response = retry_launch_eq(context.rh_launch_uac)
+    check_launch_redirect_and_token(context)
```

Comment:
We need to avoid calling steps from other steps, if the code is shared it can be factored out instead